### ISSUE TITLE
83(b)'s are increasingly not a thing

### ISSUE THREAD

**Issue by mcfunley** [jlevy/og-equity-compensation#17]
The companies I've come in contact with don't allow 83(b)'s, and I think that's been the standard advice from the law firm every VC recommends to every company for a while now. 

The guide should mention that if you think an 83(b) is appropriate, it might be something you have to try to negotiate for. (That said, it wouldn't be the first thing I'd ask for.)

**Comment by jlevy** [comment]
Yes, thanks, we should better clarify that 83(b)s may not be available or only be available if negotiated.

Not sure it's accurate to say they're going away. Whether it's available may depend a lot on stage of the company as well; my impression is they are still common for very early stage, when additional paperwork is more acceptable, and perhaps less so after that. (Does that agree with your experience?)

@joewallin may have thoughts on this.

**Comment by mcfunley** [comment]
Sounds plausible. The cases I've encountered of are all 10^1 to 10^3 employees. New companies might be more willing to make exceptions in hiring. On the other hand, they might be more easily influenced by investor advice, so I wouldn't assume they'd allow 83(b)'s without asking.

**Comment by mcfunley** [comment]
Oh, I had another thought that's possibly germane to whether or not it's "increasingly" not allowed or not. In the case of one company I worked for, they had been allowed and then were disallowed. The reason given was vague but they mentioned the [500 investor rule](http://www.investopedia.com/terms/1/500-investor-rule.asp), which was apparently a problem for Facebook at some point. 

In that case the total valuation of the company was far below $5B, so the chances of having 500 people with $10MM in holdings was remote and possibly ridiculous. But it seems as if the boilerplate in the agreements everyone is using mainly changes in reaction to the last big problem encountered by the VC class.

**Comment by rpedela** [comment]
83(b) is a form filed with the IRS on an individual basis. It is not something that can be negotiated for or against. If the employee is given shares, then they have the ability to file an 83(b). Hopefully the employer would help guide the employee to fill one out when appropriate, but that may not always be the case. However often employees are given stock options rather than shares. I am not completely sure if you can use an 83(b) with stock options, but I suspect you cannot. That would be something to look up or ask a lawyer about.

source: I am a tech startup founder and have given employees shares and guided them through the 83(b) election. I have not given stock options yet.

**Comment by ironchef** [comment]
@rpedela There's some very subtle language here.  You say "If the employee is given shares, then they have the ability to file an 83(b)."  That is correct; however, if there's a vesting period (as is usual), then this is dependent on whether or not the company allows the employee to exercise early.  If, for example, an employee is given a fairly standard 4 year vesting period with 1 year cliff, then he / she can NOT exercise stock UNLESS he / she is given rights to exercise early (in which case one must file the 83(b) within 30 days of said exercise.

Hopefully this clarifies things a bit.

Qualitative data point: most early stage companies that I know of (almost all of which are a16z) allow 83(b) for the early employees.

Source: been a part of many startups and filed 83(b) in two of them.

**Comment by rpedela** [comment]
restricted stock (shares) != stock options

I think you are talking about stock options. Like I said, I don't know all the details of 83(b) related to stock options and exercising those options. However for restricted stock via an RSPA (restricted stock purchase agreement), the company is giving the employee actual shares rather than the option to buy shares in the future. Therefore you can and should file an 83(b) immediately because the employee will lose the privilege after 30 days. If the employee leaves before the vesting term, then the company may buy back any unvested shares.

**Comment by rpedela** [comment]
To be more direct, I think this issue is invalid. 83(b) elections are not "going away". They only "go away" if the IRS and/or Congress make them go away.

**Comment by mrmcd** [comment]
The document mentions the case of options that allow you to exercise early, but with the restriction that the shares resulting from the exercise are subject to a repurchase restriction over the vesting period. Since that would presumably be a taxable event and eligible for an 83(b) election, and the whole point of an early exercise, perhaps that's what @mcfunley is referring too, rather than the 83(b) election itself?

An early exercise agreement is also something you *could* negotiate for, and also plausibly something that a lot of VCs and owners don't want to the additional complication of having to put into their option awards.

**Comment by ironchef** [comment]
@rpedela : correct...i'm talking about stock options (which are obviously handled differently).  I was inferring from the above that @jlevy might've been mixing the 83(b) with the early exercise issues (the ability if negotiated phrasing).... and you're completely correct that 83(b) is completely IRS thing.

**Comment by jlevy** [comment]
The 83(b) is between you and the IRS. But it only makes sense when you have stock that is not vested (or options that the company allows you to exercise early).

@joewallin and I have updated to reflect that a bit better. Also clarified the title of this issue.

Improved? Are we all on the same page now?

**Comment by jlevy** [comment]
Here is the updated section: https://github.com/jlevy/og-equity-compensation#83b-elections
